.gitignore

```diff
@@ -24,3 +24,5 @@ govuk_modules
 npm-debug.log
 *.DS_Store
 frontend/sass/elements
+package-lock.json
```

Comment:
https://docs.npmjs.com/files/package-lock.json

Reply:
yeah, I know that. It's a generated file which usually we shouldn't commit but yes, committing this one will solve the "but it works on my machine" issues.